diff --git a/CMakeLists.txt b/CMakeLists.txt
index 29518e1..0b930b3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -215,3 +215,5 @@ if(BUILD_TEST_DATA_GENERATORS)
   message(STATUS "Build with test data generators")
   add_subdirectory(test_data_generator)
 endif()
+
+install(DIRECTORY test_data DESTINATION share/cts)
diff --git a/test_conformance/CMakeLists.txt b/test_conformance/CMakeLists.txt
index 0b2e247..692a03a 100644
--- a/test_conformance/CMakeLists.txt
+++ b/test_conformance/CMakeLists.txt
@@ -36,3 +36,5 @@ target_link_libraries(${target} test_engine openvx-interface)
 set(TESTMODULE_FOLDER TestModule)
 
 add_subdirectory(test_module)
+
+install(TARGETS ${target} RUNTIME DESTINATION bin)
diff --git a/test_conformance/test_module/CMakeLists.txt b/test_conformance/test_module/CMakeLists.txt
index c5ea101..e48d0ab 100644
--- a/test_conformance/test_module/CMakeLists.txt
+++ b/test_conformance/test_module/CMakeLists.txt
@@ -56,7 +56,7 @@ target_link_libraries( ${TARGET_NAME_TESTMODULE} test-testmodule-lib openvx-inte
 install ( TARGETS ${TARGET_NAME_TESTMODULE_LIB} ${TARGET_NAME_TESTMODULE} ${TARGET_NAME_TESTMODULE}
           RUNTIME DESTINATION bin
           ARCHIVE DESTINATION bin
-          LIBRARY DESTINATION bin )
+          LIBRARY DESTINATION lib )
 		  
 set_target_properties( ${TARGET_NAME_TESTMODULE_LIB} PROPERTIES FOLDER ${TESTMODULE_FOLDER} )
 set_target_properties( ${TARGET_NAME_TESTMODULE} PROPERTIES FOLDER ${TESTMODULE_FOLDER} )
diff --git a/test_engine/test_utils.c b/test_engine/test_utils.c
index bc78815..829cec9 100644
--- a/test_engine/test_utils.c
+++ b/test_engine/test_utils.c
@@ -1053,8 +1053,8 @@ const char *ct_get_test_file_path()
     const char *env = getenv("VX_TEST_DATA_PATH");
     if (env == NULL)
     {
-        /* Look in the current directory */
-        return ".";
+        /* Look in the predefined directory */
+        return "/usr/share/cts/test_data/";
     }
     return env;
 }
